### DISCUSSION TITLE
Non apre il wallet

### DISCUSSION THREAD

**Discussion by neuro1978** [Chia-Network/chia-blockchain#6943] [Support]
Buongiorno , riavviato il pc ma cliccando sul programma chia blockchain apre la schermata e resta a girare senza aprire nulla , provato a disinstallare e reinstallare l'ultima versione ma niente .
Sistema operativo w10 con rizen 7 2600 32 gb ram 
![Cattura](https://user-images.githubusercontent.com/82379754/123132295-e2a60200-d44e-11eb-9f48-a7f58df11ddd.PNG)

**Comment by loppefaaret** [comment]
WINDOWS WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

Google Traduttore
RESET DEL DATABASE DEL PORTAFOGLIO DI WINDOWS

Chiudi l'app Chia e apri una finestra di PowerShell ed esegui il comando seguente:
```
mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\
```
Quindi prova ad aprire di nuovo l'app Chia
se ha funzionato, puoi rimuovere i file che abbiamo messo sul desktop!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
